updateProduct.php

```diff
@@ -0,0 +1,159 @@
+                  <!-- <span class="switch"></span> -->
+                </div>
+            </ul>
+        </div>
+    </nav>
+    <section class="dashboard">
+        <div class="top">
+            <i class="uil uil-bars sidebar-toggle"></i>
+        </div>
+        
+        <!-- Add this form below the existing top div -->
+        <div class="form-container">
+            <h2>Update Product</h2>
+
+<?php
+require 'connectdb.php';
+$id = $_GET['id'];
+$query = "SELECT * FROM products WHERE id = '$id'";
+$result = mysqli_query($conn, $query);
+$product = mysqli_fetch_assoc($result);
+mysqli_close($conn);
+?>
+
+<!-- Form to update product -->
+<form id="productForm" method="post">
+  <div class="form-row">
+    <div class="form-group">
+      <label for="productName">Product Name</label>
+      <input type="text" id="productName" name="productName" value="<?= $product['product_name'] ?>" required>
+    </div>
+    <div class="form-group">
+      <label for="category">Category</label>
+      <select id="category" name="category" required>
+        <option value="" disabled selected>Select Category</option>
+        <?php
+        require 'connectdb.php';
+        $query = "SELECT * FROM categories";
+        $result = mysqli_query($conn, $query);
+        while ($category = mysqli_fetch_assoc($result)) {
+          ?>
+          <option value="<?= $category['id'] ?>" <?= $product['category_id'] == $category['id'] ? 'selected' : '' ?>><?= $category['category_name'] ?></option>
+          <?php
+        }
+        mysqli_close($conn);
+        ?>
+      </select>
+    </div>
+  </div>
+  <div class="form-row">
+    <div class="form-group">
+      <label for="price">Price</label>
+      <input type="number" id="price" name="price" value="<?= $product['price'] ?>" required>
+    </div>
+    <div class="form-group">
+      <label for="quantity">Quantity</label>
+      <input type="number" id="quantity" name="quantity" value="<?= $product['quantity'] ?>" required>
+    </div>
+  </div>
+  <div class="form-row">
+    <div class="form-group">
+      <label for="description">Description</label>
+      <textarea id="description" name="description" rows="4" required><?= $product['description'] ?></textarea>
+    </div>
+  </div>
+  <button type="submit">Update Product</button>
+</form>
+
+        </div>
+    </section>
+    
+    <?php
+require 'connectdb.php';
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+//   $id = $_GET["id"];
+  $productName = $_POST["productName"];
+  $category = $_POST["category"];
+  $price = $_POST["price"];
+  $quantity = $_POST["quantity"];
+  $description = $_POST["description"];
+
+  $query = "UPDATE products SET product_name = '$productName', category_id = '$category', price = '$price', quantity = '$quantity', description = '$description' WHERE id = '$id'";
+  mysqli_query($conn, $query);
+
+  mysqli_close($conn);
+
+  header("Location: products.php");
+  exit;
+}
+?>
+
+
+    <script src="script.js"></script>

```

Comment:
This needs to be used above. As it is not a good practice to use this here. Update this

Reply:
Updated the code and resolved that issue